### ISSUE TITLE
PlayerGlow failing to inititialize.

### ISSUE THREAD

**Issue by Firav** [MrHDR/Player_Glow_Sponge#12]
Here are the errors I am encountering. Spamming console with Sponge errors.

https://pastebin.com/5K7EQcHm

https://pastebin.com/w9Hd214a

Plugin version: 1.3.0

Running the FTB Revelation modpack

MC: 1.12.2
SpongeAPI: 7.1.10-240a43e70
Sponge: 1.12.2-7.1.7-SNAPSHOT
SpongeForge: 1.12.2-2838-7.1.7-RC3834
MC Forge: 14.23.5.2836

**Comment by HDR** [comment]
Yeah, this seems to be an issue on newer sponge builds, i will have a look into it when i got some time

**Comment by Firav** [comment]
Version 1.1.1 seems to work fine, just cant change colors.
